extension/data/deinflect.dat

```diff
@@ -363,3 +360,6 @@ past negative
 でございましょう	でございます	8320	18
 でございまして	でございます	8320	21
 であれば	でございます	8320	17
+いって	いく	384	0
+いった	いく	384	1
```

Comment:
The 384 here might be wrong. It looks like it corresponds to a hex value of 0x0180 which means a deinflection of OTHER to RU VERB (See https://gist.github.com/lynn/07f7ce3c314223d2aca19ec2bb0540cd). But in this case iku is more like a U VERB than a RU VERB right?

In that case it seems like we should use 0x0280 which is 640 decimal.

In that way it looks like we probably can reuse the values from the previous lines here as well.

This is also could have been why 行って stopped working. Since it wouldn't have recognized it as a RU verb.

Reply:
I did those changes but apparently the 行って problem still persists for some reason.

Reply:
Nothing is obviously wrong so I guess the next step is to step through the relevant lookup code in data.ts with devtools and figure out why it's not matching. I'll try to find some time today or tomorrow to checkout your PR and debug, though feel free to try yourself. :)

Reply:
Alright, I debugged it a bit and there were two problems. 

1. I forgot how simple the logic for this is. It doesn't actually convert kanji to kana for the deinflection.  So we need to actually add the kanji to the file like:
```
行って	行く	640	21
行った	行く	640	14
行っちゃう	行く	514	3
```

This works well but means that if someone is looking up the kana form it won't match. We could add the kana form as well but then words like 往く will also match. I feel like avoiding wrong results is better than not having every corner case  covered since 行く is usually written with the kanji.

2. There was a small typo in the いっちゃう entry where you used a space instead of a tab. I only saw it after I tested the above fix and it still didn't work. ^_^;

Reply:
Now looks like it's working, thank you for helping me and sorry for bothering you with a lot of questions. This was actually my first time doing open source.